profile/src/v100/entity/persona/persona.rs

```diff
@@ -120,36 +114,32 @@ impl Persona {
         let private_hd_factor_source =
             PrivateHierarchicalDeterministicFactorSource::new(mwp, bdfs);
 
-        let ctr = Arc::<Mutex<u64>>::new(Mutex::new(1));
-        let next = || {
-            let n = Uuid::from_u64_pair(0, *ctr.lock().unwrap().borrow());
-            ctr.lock().unwrap().borrow_mut().add_assign(1);
-            n
-        };
-
+        let id = IDStepper::<PersonaDataEntryID>::new();
+        let name =
+            PersonaDataIdentifiedName::with_id(unsafe { id.next() }, name);
```

Comment:
`unsafe` usage is actually `safe`?

Reply:
Right, it is incorrect usage of `unsafe` in Rust, as it does not relate to memory safety. But since this is not going to be used in Production code (apart from Placeholder / Preview value generation) I'd reckon it was a convenient, albeit incorrect, way of introducing friction of accidentally using the `IDStepper` and `next` method.

I cannot gate it behind `#[cfg(test]` since Placeholder is not gated by `#[cfg(test)]`, since we want to be able to expose the placeholders in Swift and Android as well...